be/test/exec/test_data/json_scanner/test_debezium_json_load.json

```diff
@@ -0,0 +1,246 @@
+[
```

Comment:
Have you considered the json format including `schema` and `payload` as [debezium documentation](https://debezium.io/documentation/reference/stable/connectors/mysql.html) descibes.